### PR TITLE
Restructuring CI Folder && PHP CS Fixer

### PR DESCRIPTION
**Pull request by gvollbach**

### CODE REVIEW

**Review by alex40724 — APPROVED** [review]

**Comment by alex40724** [comment]
Looks great. Thanks!

File: CI/PHP-CS-Fixer/example/ILIAS_CodeStyle.xml [hunk 1]
```diff
@@ -0,0 +1,333 @@
+<code_scheme name="ILIAS Code Style" version="173">
```
Comment (legionth) [inline CI/PHP-CS-Fixer/example/ILIAS_CodeStyle.xml]: Maybe we should rename the file or create a sub-directory to identify this is a file for `PHPStorm`

**Review by legionth — CHANGES_REQUESTED** [review]

File: CI/PHP-CS-Fixer/example/ILIAS_CodeStyle.xml (hunk 1 above)
Comment (gvollbach) [reply CI/PHP-CS-Fixer/example/ILIAS_CodeStyle.xml]: @legionth & @mjansenDatabay any concrete suggestions for folder or file name?

Comment (legionth) [reply CI/PHP-CS-Fixer/example/ILIAS_CodeStyle.xml]: `CI/PHP-CS-Fixer/example/PhpStorm/ILIAS_CodeStyle.xml`

`CI/PHP-CS-Fixer/example/PhpStorm_ILIAS_CodeStyle.xml`

?

Comment (legionth) [reply CI/PHP-CS-Fixer/example/ILIAS_CodeStyle.xml]: I'm preferring the first.

**Review by legionth — APPROVED** [review]

File: CI/PHP-CS-Fixer/example/ILIAS_CodeStyle.xml (hunk 1 above)
Comment (gvollbach) [reply CI/PHP-CS-Fixer/example/ILIAS_CodeStyle.xml]: Me too, done.

Comment (mjansenDatabay) [reply CI/PHP-CS-Fixer/example/ILIAS_CodeStyle.xml]: Option 1 please.

**Review by Amstutz — APPROVED** [review]

**Review by klees — APPROVED** [review]
LGTM, thx Guido!

**Pull request merged**